tensorflow/python/kernel_tests/split_op_test.py

```diff
@@ -153,6 +153,14 @@ def testGradientsAll(self):
     self._testGradientsSimple(use_gpu=False)
     self._testGradientsSimple(use_gpu=True)
 
+  def testOutputShape(self):
```

Comment:
Looks great, thank you, but the indentation is not consistent here with the rest of the file.  Please change to 2 spaces.